config/initializers/friendly_id.rb

```diff
@@ -0,0 +1,92 @@
+# FriendlyId Global Configuration
+#
+# Use this to set up shared configuration options for your entire application.
+# Any of the configuration options shown here can also be applied to single
+# models by passing arguments to the `friendly_id` class method or defining
+# methods in your model.
+#
+# To learn more, check out the guide:
+#
+# http://norman.github.io/friendly_id/file.Guide.html
+
+FriendlyId.defaults do |config|
+  # ## Reserved Words
+  #
+  # Some words could conflict with Rails's routes when used as slugs, or are
+  # undesirable to allow as slugs. Edit this list as needed for your app.
+  config.use :reserved
+
+  config.reserved_words = %w(new edit index session login logout users admin
```

Comment:
seems like this isn't working properly. When I navigate to `http://localhost:3000/repositories/index` I get:

<img width="500" alt="screen shot 2017-06-22 at 8 53 38 am" src="https://user-images.githubusercontent.com/1131894/27443571-5e0bfb32-5728-11e7-8823-c8ee3249fb8e.png">